openstates/tn/bills.py

```diff
@@ -158,7 +158,7 @@ def categorize_action(action):
     return list(types), attrs
 
 
-def actions_from_table(bill, actions_table):
+def actions_from_table(self, bill, actions_table):
```

Comment:
looks good to me, but curious if it'd make sense to just move this into the class body now that it takes self?

Reply:
Yeah this makes sense. I also see a stray print statement I left in --  I'll take another look this afternoon or tomorrow and maybe refactor it to be a little cleaner.